distributed/protocol/serialize.py

```diff
@@ -572,19 +574,21 @@ class ToPickle:
     to False, the scheduler will raise an exception instead.
     """
 
-    def __init__(self, data):
+    data: T
+
+    def __init__(self, data: T) -> None:
         self.data = data
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return "<ToPickle: %s>" % str(self.data)
 
-    def __eq__(self, other):
+    def __eq__(self, other: object) -> bool:
         return isinstance(other, type(self)) and other.data == self.data
 
-    def __ne__(self, other):
+    def __ne__(self, other: object) -> bool:
         return not (self == other)
```

Comment:
out of scope: fairly sure this method is redundant with `__eq__`

Reply:
Good point, didn't even pay attention to this. Dropped `__ne__` as a drive-by fix.